### ISSUE TITLE
as3mxml.sdk.framework possibly could use relative paths?

### ISSUE THREAD

**Issue by cgascons** [BowlerHatLLC/vscode-as3mxml#523]
Hi, simple question here, not really an issue. Is there a way where `as3mxml.sdk.framework` could accept relative paths? We are a 2-man team and the path for our sdk's change from user to user, for example my current sdk path is the following: 

```
"settings": {
	"as3mxml.sdk.framework": "/Users/cgascons/workspaceAIR/_SDKS/air_33_259" //could it be changed to: "../../_SDKS/air_33_259"?
}
```

while my workmate varies on the username.
Thanks in advance!

**Comment by joshtynjala** [comment]
It's interesting that you share the same settings.json file between multiple people. I always make sure to exclude the files in .vscode/ from being committed to repos to ensure that everyone can set up their environment how they want.

If I find some time, I'll see if this is possible. Could be tricky, though. Especially in workspaces with multiple root folders.

**Comment by cgascons** [comment]
Thanks! No rush at all on our end, just thought I'd ask in case it was something already implemented.

**Comment by varadig** [comment]
it would be really usefull for me too

**Comment by varadig** [comment]
what about this solution
https://github.com/BowlerHatLLC/vscode-as3mxml/pull/555

**Comment by joshtynjala** [comment]
Closed by #555
